writeit/settings.py

```diff
@@ -67,6 +67,10 @@
 # If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
+# If the site is being served over https change this to 'https' so that links
+# to subdomains use the correct scheme.
+DEFAULT_URL_SCHEME = 'http'
+
```

Comment:
I think this is only relevant for the django-subdomains' `urljoin` function, which isn't used in write-it or in the rest of django-subdomains, so I probably wouldn't bother with this.